lib/common/watchdog.c

```diff
@@ -26,8 +26,7 @@
 
 static pid_t sbd_pid = 0;
 
-enum pcmk_panic_flags
-{
+enum pcmk_panic_flags {
     pcmk_panic_none     = 0x00,
     pcmk_panic_delay    = 0x01,
     pcmk_panic_kdump    = 0x02,
```

Comment:
What's this enum used for?  I can't find any references to it nor to any of its members in the source tree.

Reply:
Yep looks like it was never used, good catch :)